example/session/src/main/java/org/apache/iotdb/SessionExample.java

```diff
@@ -247,15 +247,11 @@ private static void insertTablet() throws IoTDBConnectionException, StatementExe
 
     Tablet tablet = new Tablet("root.sg1.d1", schemaList, 100);
 
-    long[] timestamps = tablet.timestamps;
-    Object[] values = tablet.values;
-
     for (long time = 0; time < 100; time++) {
-      int row = tablet.rowSize++;
-      timestamps[row] = time;
-      for (int i = 0; i < 3; i++) {
-        long[] sensor = (long[]) values[i];
-        sensor[row] = i;
+      int rowIndex = tablet.rowSize++;
+      tablet.addTimestamp(rowIndex, time);
+      for (int s = 0; s < 3; s++) {
```

Comment:
I think it's better to separate the rowIndex, the time column, and the value column, which would clearer to users.

Reply:
copy that,thank you